src/gc.c

```diff
@@ -378,17 +377,17 @@ void runGC(const time_t now, time_t *lastGCrun, const bool flush)
 
 		// Update reply counters
 		counters->reply[query->reply]--;
-		log_debug(DEBUG_GC, "reply type %d removed (GC), ID = %d, new count = %d", query->reply, query->id, counters->reply[query->reply]);
+		log_debug(DEBUG_STATUS, "reply type %d removed (GC), ID = %d, new count = %d", query->reply, query->id, counters->reply[query->reply]);
```

Comment:
Why did you change all the instances from ` DEBUG_GC > DEBUG_STATUS`?

Here it even talks about the garbage collection.

Reply:
Because the status debug flag is meant specifically to log every single counter change (for every query!) so it's very very verbose. The GC debug flag prints much fewer lines with more condensed content. As these lines make no sense without DEBUG_STATUS being set at the same time (for context), I moved them over there